openstef/app_settings.py

```diff
@@ -9,9 +9,38 @@
 class AppSettings(BaseSettings):
     """Global app settings."""
 
-    # Logging settings.
-    log_level: str = Field("INFO", description="Log level used for logging statements.")
-
     model_config = SettingsConfigDict(
         env_prefix="openstef_", env_file=".env", extra="ignore"
     )
+
+    # Weather column names
+    weather_column_name_temperature: str = "temp"
+    weather_column_name_radiation: str = "radiation"
+    weather_column_name_windspeed: str = "windspeed"
+    weather_column_name_windspeed_100m: str = "windspeed_100m"
+    weather_column_name_windspeed_100m_extrapolated: str = "windspeed_100mExtrapolated"
+    weather_column_name_saturation_pressure: str = "saturation_pressure"
+    weather_column_name_vapour_pressure: str = "vapour_pressure"
+    weather_column_name_dewpoint: str = "dewpoint"
+    weather_column_name_air_density: str = "air_density"
+    weather_column_name_humidity: str = "humidity"
+    weather_column_name_pressure: str = "pressure"
+    weather_column_name_wind_power_fit_extrapolated: str = "windPowerFit_extrapolated"
+    weather_column_name_wind_power_fit_harm_arome: str = "windpowerFit_harm_arome"
+    weather_column_name_wind_hub_height: str = "hub_height"
+
+    weather_column_name_turbine_type: str = "turbine_type"
+    weather_column_name_number_turbines: str = "n_turbines"
+
+    location_column_name_latitude: str = "lat"
+    location_column_name_longitude: str = "lon"
```

Comment:
For me this is more explicit than needed. Lat and Lon are pretty common as-is, so as long as OpenSTEF is clear it needs this, I don't think its needs to be a custom setting? 
Also, are these terms not already defined in the Prediction Job the user needs to input?